exercises/03.async/01.solution.await/README.mdx

```diff
@@ -1,4 +1,6 @@
-# Awaiting promises
+# Awaiting Async Tests
```

Comment:
This puts emphasis that the _test_ is async while the exercise is about async _code_ we are testing. 

I think your rewrite to "Handling Promises in Tests" would still make perfect sense for this title too!